modules/tensor_mechanics/src/materials/RecomputeRadialReturn.C

```diff
@@ -140,6 +140,19 @@ RecomputeRadialReturn::getIsotropicShearModulus()
   return shear_modulus;
 }
 
+Real
+RecomputeRadialReturn::getIsotropicBulkModulus()
+{
+  Real shear_modulus = getIsotropicShearModulus();
+  // dilatational modulus is defined as lambda plus two mu
+  Real dilatational_modulus = _elasticity_tensor[_qp](0,0,0,0);
+  if (_mesh.dimension() == 3 && dilatational_modulus != _elasticity_tensor[_qp](2,2,2,2))
```

Comment:
This check is potentially run a lot of times (even if the underlying elasticity tensor does not change during a recompute loop). I wonder if we should put this someplace else (like an element user object for example) so that we can run it at the desired times (timestep_begin, linear, etc.)